Test `test_disks_app_func` is broken

Seems like it depends on tests execution order and the order changed recently, so test stopped working:
```
    def test_disks_app_func_ls(started_cluster):
        source = cluster.instances["disks_app_test"]
    
        init_data(source)
    
        out = source.exec_in_container(
            ["/usr/bin/clickhouse", "disks", "--send-logs", "--disk", "test1", "list", "."]
        )
    
        files = out.split("\n")
    
>       assert files[0] == "store"
E       AssertionError: assert 'path1' == 'store'
E         - store
E         + path1
```
https://play.clickhouse.com/play?user=play#c2VsZWN0IAp0b1N0YXJ0T2ZIb3VyKGNoZWNrX3N0YXJ0X3RpbWUpIGFzIGQsCmNvdW50KCksICBncm91cFVuaXFBcnJheShwdWxsX3JlcXVlc3RfbnVtYmVyKSwgIGFueShyZXBvcnRfdXJsKQpmcm9tIGNoZWNrcyB3aGVyZSAnMjAyMi0wNi0wMScgPD0gY2hlY2tfc3RhcnRfdGltZSBhbmQgdGVzdF9uYW1lIGxpa2UgJyV0ZXN0X2Rpc2tzX2FwcF9mdW5jJScgYW5kIHRlc3Rfc3RhdHVzIGluICgnRkFJTCcsICdGTEFLWScpIGdyb3VwIGJ5IGQgb3JkZXIgYnkgZCBkZXNj

cc: @Varinara, @kssenii

--- a/tests/integration/test_disks_app_func/test.py
+++ b/tests/integration/test_disks_app_func/test.py
@@ -130,9 +130,27 @@ def test_disks_app_func_cp(started_cluster):
             "path1",
         ]
     )
+    
+    source.exec_in_container(
+        [
+            "/usr/bin/clickhouse",
+            "disks",
+            "--send-logs",
+            "--disk",
+            "test1",
+            "remove",
+            "path1",
+        ]
+    )
 
     out = source.exec_in_container(
         ["/usr/bin/clickhouse", "disks", "--send-logs", "--disk", "test2", "list", "."]
+    )
+
+    assert "path1" not in out
+    
+    out = source.exec_in_container(
+        ["/usr/bin/clickhouse", "disks", "--send-logs", "--disk", "test1", "list", "."]
     )
 
     assert "path1" not in out

--- a/tests/integration/test_disks_app_func/test.py
+++ b/tests/integration/test_disks_app_func/test.py
@@ -143,6 +143,8 @@ def test_disks_app_func_cp(started_cluster):
         ]
     )
 
+    # alesapin: Why we need list one more time?
+    # kssenii: it is an assertion that the file is indeed deleted
     out = source.exec_in_container(
         ["/usr/bin/clickhouse", "disks", "--send-logs", "--disk", "test2", "list", "."]
     )

--- a/tests/integration/test_disks_app_func/test.py
+++ b/tests/integration/test_disks_app_func/test.py
@@ -118,6 +118,24 @@ def test_disks_app_func_cp(started_cluster):
     )
 
     assert "path1" in out
+    
+    source.exec_in_container(
+        [
+            "/usr/bin/clickhouse",
+            "disks",
+            "--send-logs",
+            "--disk",
+            "test2",
+            "remove",
+            "path1",
+        ]
+    )
+
+    out = source.exec_in_container(
+        ["/usr/bin/clickhouse", "disks", "--send-logs", "--disk", "test2", "list", "."]
+    )
+
+    assert "path1" not in out
 
 
 def test_disks_app_func_ln(started_cluster):

--- a/tests/integration/test_disks_app_func/test.py
+++ b/tests/integration/test_disks_app_func/test.py
@@ -118,7 +118,7 @@ def test_disks_app_func_cp(started_cluster):
     )
 
     assert "path1" in out
-    
+
     source.exec_in_container(
         [
             "/usr/bin/clickhouse",

--- a/tests/integration/test_disks_app_func/test.py
+++ b/tests/integration/test_disks_app_func/test.py
@@ -130,7 +130,7 @@ def test_disks_app_func_cp(started_cluster):
             "path1",
         ]
     )
-    
+
     source.exec_in_container(
         [
             "/usr/bin/clickhouse",
@@ -148,7 +148,7 @@ def test_disks_app_func_cp(started_cluster):
     )
 
     assert "path1" not in out
-    
+
     out = source.exec_in_container(
         ["/usr/bin/clickhouse", "disks", "--send-logs", "--disk", "test1", "list", "."]
     )

